src/meta/types/src/user_stage.rs

```diff
@@ -296,36 +298,76 @@ pub struct StageParams {
 
 #[derive(serde::Serialize, serde::Deserialize, Clone, Debug, Eq, PartialEq)]
 pub enum OnErrorMode {
-    None,
```

Comment:
Does `OnErrorMode` stored on the meta-service when creating stage? If so, this is a breaking change.

Reply:
This has been stored before, this PR just adds a type `AbortNum(n)`